custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -576,18 +648,21 @@ function DownloadController($scope, $rootScope, $location, locationHierarchy, lo
     vm.showViewBy = function () {
         return !(vm.isChildBeneficiaryListSelected() || vm.isIncentiveReportSelected() ||
             vm.isLadySupervisorSelected() || vm.isDashboardUsageSelected() ||
-            vm.isChildGrowthTrackerSelected() || vm.isTakeHomeRationReportSelected() ||
-            vm.isAwwActivityReportSelected());
+            vm.isChildGrowthTrackerSelected() || vm.isTakeHomeRationReportSelected() || vm.isAwwActivityReportSelected());
     };
 
     vm.showLocationFilter = function () {
         return !vm.isDashboardUsageSelected();
     };
 
     vm.showMonthFilter = function () {
-        return !(vm.isDashboardUsageSelected() || vm.isAwwActivityReportSelected());
+        return !(vm.isDashboardUsageSelected() || vm.isAwwActivityReportSelected() || vm.isQuarterDataPeriodSelected());
```

Comment:
should this be `!(vm.isDashboardUsageSelected() || (vm.isAwwActivityReportSelected() && vm.isQuarterDataPeriodSelected()))`

Reply:
both are three separate conditions, so or for all of them, if we put && then it would not work perfectly